templates/studentAffectedByModule.html

```diff
@@ -0,0 +1,16 @@
+$def with (moduleCode, aysem, student_lst)
```

Comment:
Ok, but please add a page description that will mention what does **affected** mean in this page's context.